tests/NLog.UnitTests/LayoutRenderers/CallSiteTests.cs

```diff
@@ -31,6 +31,9 @@
 // THE POSSIBILITY OF SUCH DAMAGE.
 // 
 
+using System.IO;
+using System.Threading.Tasks;
+
```

Comment:
I just noticed Visual Studio's "Quick Actions" adds the `using` inside the `namespace` (if there are no `using`s outside of it). Interesting.